### ISSUE TITLE
AttributeError: 'list' object has no attributes 'values'

### ISSUE THREAD

**Issue by qalshidi** [NiklasRosenstein/pydoc-markdown#149]
Hi,

It seems there is an error when I try to use pydoc-markdown, I'm unsure what is causing it or how to advise reproducing it:

```
[INFO - pydoc_markdown]: Loading modules.
[INFO - pydoc_markdown.contrib.loaders.python]: Load Python modules (search_path: ['./.', '/usr/lib/python3.8', '/usr/lib/python3.8/lib-dynload/', '/usr/lib/python3.8/site-packages/', '/home/qusai/.local/lib/python3.8/site-packages/'], modules: ['swmfpy', 'swmfpy.web', 'swmfpy.io', 'swmfpy.paramin', 'swmfpy.tools', 'swmfpy.tecplottools'], packages: [], discover: False)
[WARNING - pydoc_markdown.contrib.processors.crossref]: 4 cross-reference(s) could not be resolved:
  swmfpy.write_imf_from_omni: swmfpy.io.write_imf_input, swmfpy.web.get_omni_data
  swmfpy.tecplottools.interpolate_zone_to_geometry: SATELLITE, SATELLITE
Traceback (most recent call last):
  File "/home/qusai/.local/bin/pydoc-markdown", line 33, in <module>
    sys.exit(load_entry_point('pydoc-markdown==3.3.0', 'console_scripts', 'pydoc-markdown')())
  File "/home/qusai/.local/lib/python3.8/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/home/qusai/.local/lib/python3.8/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/home/qusai/.local/lib/python3.8/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/qusai/.local/lib/python3.8/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/home/qusai/.local/lib/python3.8/site-packages/pydoc_markdown/main.py", line 336, in cli
    session.render(pydocmd)
  File "/home/qusai/.local/lib/python3.8/site-packages/pydoc_markdown/main.py", line 136, in render
    config.render(modules)
  File "/home/qusai/.local/lib/python3.8/site-packages/pydoc_markdown/__init__.py", line 160, in render
    self.renderer.render(modules)
  File "/home/qusai/.local/lib/python3.8/site-packages/pydoc_markdown/contrib/renderers/markdown.py", line 393, in render
    self._render_modules(modules, self.fp or sys.stdout)
  File "/home/qusai/.local/lib/python3.8/site-packages/pydoc_markdown/contrib/renderers/markdown.py", line 308, in _render_modules
    self._render_toc(fp, 0, m)
  File "/home/qusai/.local/lib/python3.8/site-packages/pydoc_markdown/contrib/renderers/markdown.py", line 200, in _render_toc
    for child in obj.members.values():
AttributeError: 'list' object has no attribute 'values'
```

**Comment by NiklasRosenstein** [comment]
Thanks for reporting @qalshidi

**Comment by NiklasRosenstein** [comment]
Fixed in 3.3.1
